frontend/jest.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'jsdom',
+  moduleNameMapper: {
```

Comment:
we're going to need this in the future for modules that import css. otherwise our unit tests will fail